sympy/geometry/line.py

```diff
@@ -1070,6 +1071,48 @@ def contains(self, other):
             return Point.is_collinear(self.p1, self.p2, other.p1, other.p2)
         return False
 
+    def parameter_value(self, other):
+        """
+        Finds the parameter ``t`` for which
```

Comment:
`t` and `1 - t` are usually known as [barycentric coordinates](https://en.wikipedia.org/wiki/Barycentric_coordinate_system) of the point `x`. I think that something like `coordinate` would be a better name for the function.

Reply:
Sure ,will do that.

Reply:
It looks like the note could be removed.